x-pack/filebeat/module/aws/elb/ingest/pipeline.yml

```diff
@@ -89,17 +89,58 @@ processors:
         ELBV2TYPE: '%{WORD:aws.elb.type}'
         ELBV2LOGVERSION: '%{NOTSPACE}'  # Could be used to support different log versions, only 1.0 exists now
 
+  - set:
+      field: event.kind
+      value: event
+
+  - set:
+      field: cloud.provider
+      value: aws
 
   - set:
       if: 'ctx.http != null'
       field: 'aws.elb.protocol'
       value: 'http'
 
+  - set:
+      if: 'ctx.http != null'
+      field: event.category
+      value: web
+
   - set:
       if: 'ctx.http == null'
       field: 'aws.elb.protocol'
       value: 'tcp'
 
+  - set:
+      if: 'ctx.http == null'
+      field: event.category
+      value: network
+
+  - convert:
+      field: http.response.status_code
```

Comment:
This can be accomplished in the grok statement by adding `:long` like we have in `%{NUMBER:http.request.body.bytes:long}`.

Reply:
Using the grok should also be safer, because if this fails with `ignore_failure` the data won't be indexed because of a mapping exception (assuming the failed is mapped as a long).